src/test/java/iudx/aaa/server/policy/ListPolicyNotificationTest.java

```diff
@@ -308,6 +308,37 @@ void failNotRegisteredUser(VertxTestContext testContext) {
   @DisplayName("Test admin/trustee calling API")
   void failDisallowedRoles(VertxTestContext testContext) {
     // same as the create notification tests here
+    Checkpoint checkAdmin = testContext.checkpoint();
+    Checkpoint checkTrustee = testContext.checkpoint();
+    JsonObject admin = consumer.result();
+    String randomUserId = UUID.randomUUID().toString();
+    User userAdmin = new UserBuilder().keycloakId(admin.getString("keycloakId")).userId(randomUserId)
+            .name(admin.getString("firstName"), admin.getString("lastName"))
+            .roles(List.of(Roles.ADMIN)).build();
+
+    policyService.listPolicyNotification(userAdmin, new JsonObject(),
+            testContext.succeeding(response -> testContext.verify(() -> {
+              assertEquals(URN_INVALID_ROLE.toString(), response.getString(TYPE));
+              assertEquals(ERR_DETAIL_LIST_DELEGATE_ROLES, response.getString("detail"));
+              assertEquals(ERR_TITLE_INVALID_ROLES, response.getString("title"));
+              assertEquals(401, response.getInteger("status"));
+              checkAdmin.flag();
+            })));
+
+    JsonObject trustee = consumer.result();
+    User trusteeUser = new UserBuilder().keycloakId(trustee.getString("keycloakId")).userId(randomUserId)
+            .name(trustee.getString("firstName"), trustee.getString("lastName"))
+            .roles(List.of(Roles.TRUSTEE)).build();
+
+    policyService.listPolicyNotification(trusteeUser, new JsonObject(),
+            testContext.succeeding(response -> testContext.verify(() -> {
+              assertEquals(URN_INVALID_ROLE.toString(), response.getString(TYPE));
+              assertEquals(ERR_DETAIL_LIST_DELEGATE_ROLES, response.getString("detail"));
+              assertEquals(ERR_TITLE_INVALID_ROLES, response.getString("title"));
+              assertEquals(401, response.getInteger("status"));
+              checkTrustee.flag();
+            })));
+
     testContext.completeNow();
```

Comment:
You can remove this line - the `testContext` will complete when all checkpoints are flagged, so no need to have this line too.